packages/components/src/responsive-wrapper/stories/index.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import ResponsiveWrapper from '..';
+
+const meta: ComponentMeta< typeof ResponsiveWrapper > = {
```

Comment:
Should we hide `children`'s control? It currently shows a react object, which is not really user-friendly to edit